app/Rom.php

```diff
@@ -1139,6 +1139,11 @@ public function setGanonInvincible(string $setting = 'no'): self
      */
     public function setHeartColors(string $color): self
     {
+        if ($color === 'random') {
+          $colorOptions = ['blue', 'green', 'yellow', 'red'];
+          $color = $colorOptions[mt_rand(0, 3)];
+        }
```

Comment:
I'm fine with this as a PR, however I'm not particular on adding randomization into the Rom class, it breaks a separation of concerns, it should only know how to set things you request, I would expect the logic to be at the JS/Controller/Command level

Reply:
Extracted! Shame the class calling this one doesn't have unit tests though (yet!). Also `random_int()` doesn't appear to be seed-able so I can't really fake it anyway. (I'm not too familiar with fakes/mocks/etc. in PHP yet; my first foray into this in many moons.)